sorc/ncep_post.fd/WRFPOST.f

```diff
@@ -444,6 +444,26 @@ PROGRAM WRFPOST
             print*,'error opening ',fileName, ' Status = ', Status 
             stop
           endif
+          Status = nf90_open(trim(fileNameFlux),NF90_NOWRITE, ncid2d)
+          if ( Status /= 0 ) then
+            print*,'error opening ',fileNameFlux, ' Status = ', Status
+            stop
+          endif
+! read in LSM index and nsoil here
+          Status=nf90_get_att(ncid2d,nf90_global,'landsfcmdl', &
+                 iSF_SURFACE_PHYSICS)
+          if(Status/=0)then
+            print*,'landsfcmdl not found; assigning to 2'
+            iSF_SURFACE_PHYSICS=2 !set LSM physics to 2 for NOAH
+          endif
+          Status=nf90_get_att(ncid2d,nf90_global,'nsoil', &
+                 NSOIL)
```

Comment:
cosmetic suggestion, it looks like line 459 and 460 can be put into one line. Similar for Line 453,454.

Reply:
Thanks, good idea.  I'll make that change.